src/Symfony/Bundle/WebProfilerBundle/DependencyInjection/WebProfilerExtension.php

```diff
@@ -54,6 +54,14 @@ public function load(array $configs, ContainerBuilder $container)
             $container->setParameter('web_profiler.debug_toolbar.mode', $config['toolbar'] ? WebDebugToolbarListener::ENABLED : WebDebugToolbarListener::DISABLED);
         }
 
+        if ($config['intercept_redirects']) {
+            @trigger_error(
```

Comment:
Can you put the everything on one line?

Reply:
shouldnt deprecate using the Configuration tooling?